src/computed/notification.js

```diff
@@ -22,6 +28,9 @@ const ComputedNotification = store => {
       });
       return all;
     }),
+    notificationCountLabel: computed(() =>
+      formatNumber(store.computedNotifications.length)
+    ),
```

Comment:
Also moved this here to be computed based on the `computedNotifications` list size.